grants/milestone-delivery-guidelines.md

```diff
@@ -0,0 +1,27 @@
+# Milestone Deliverables Guidelines
+
+We kindly ask you to consider the following guidelines when you submit a milestone for evaluation. Please submit your milestones by email to grants@web3.foundation.
+
+The submission should contain the following information: 
+ * **Name of the grant project**
+ * **Link to the open-source code/delivery** 
+ * **Basic [testing guide](#testing-guide)** 
+ * **A list of the [milestone deliverables](#milestone-deliverables)**
+ * **Any [additional information](#additional-information)**
+
+## Testing Guide
+
+Please provide documentation on how to install, compile and/or test the delivery. Ideally, this is done in the form of a publicly available readme file or something similiar, so that others can potentially leverage the project and learn from it. For the installation, we prefer docker images to avoid issues with versions and dependencies.
+
+## Milestone Deliverables
+
+Please provide a list of all deliverables of the milestone extracted from the initial application (see Annex 1 of the Agreement) and a link to the deliverable itself. Add additional notes to the deliverable, especially in case it deviates from the initial contract. Furthermore, please ensure the repo has the correct open-source license.
```

Comment:
New Wording:

"Please provide a list of milestone deliverables. This list should closely reflect the list of deliverables shown in Annex 1 of the grant contract. Each item in the list should include a link to the deliverable itself. If necessary, please highlight anything that deviates from the contract and include further information that you think is relevant to deliverable.

Please ensure the repo has the correct open-source license."

Reply:
Updated it. Thanks